py/server/deephaven/parquet.py

```diff
@@ -69,6 +69,8 @@ def _build_parquet_instructions(
     base_name: Optional[str] = None,
     file_layout: Optional[ParquetFileLayout] = None,
     table_definition: Optional[Union[Dict[str, DType], List[Column]]] = None,
+    col_definitions: Optional[List[Column]] = None,
```

Comment:
I think it would be better to force the `write` path to convert to `TableDefinition` before calling this function.